wiki/Beatmap_Editor/AiMod/en.md

```diff
@@ -2,12 +2,22 @@
 
 **AiMod** is a modding tool built into the [Beatmap Editor](/wiki/Beatmap_Editor). To open AiMod, press `Control` + `Shift` + `A` or navigate via menu options **File** » **Open AiMod** once a beatmap is open in the Beatmap Editor.
 
-AiMod aims to help beginner mappers "mod" their beatmaps. The limitation of AiMod lies with its basic design which causes it to sometimes overlook issues such as bad mapping patterns, BPM, or timing. Finding a [modder](/wiki/Glossary) to mod your beatmap is strongly recommended.
+AiMod aims to help mappers "mod" their beatmaps. The limitation of AiMod lies with its basic design which causes it to sometimes overlook issues such as bad mapping patterns and incorrect timing. Finding a [modder](/wiki/Glossary) to mod your beatmap is strongly recommended.
 
-![AiMod](AiMod.jpg "AiMod is usually your first modder")
+![AiMod](AiMod.jpg "This is the AiMod window")
 
 AiMod typically completes its beatmap analysis in under a minute and subsequently displays warnings and errors. If no problems were found in your beatmap, then congratulations! The beatmap passed the AiMod analysis, but it still requires modding from the community. The modders can find other issues that AiMod failed to detect.
 
+## Flaws
+
+<!-- TO DO: I'm sure there are more flaws, just can't think of any more off the top of my head -->
+- **Cannot detect a large number of issues** that are considered unrankable by the [Ranking Criteria](/wiki/Ranking_Criteria), such as; potentially missnapped objects, delayed hitsounds and background filesize above 2.5MB etc.
+- **May warn you of non-issues**, such as; the mapset needing 2 standard diffs on a marathon map and Kiai time being toggled on for less than 15 seconds.
+- **If "Check distance snap" is enabled, it will not check the distance snap between new combos.**
```

Comment:
this is intentional, it checks per-combo

Reply:
hmm was thinking of it as a flaw from the mapper's perspective, since aimod doesn't detect spacing inconsistencies between combos in normal/easy diffs (which it probably should) which leads into being one of the reasons to use mapset verifier to check these things

but if we want to stick with unintentional flaws of aimod i can remove this one

Reply:
maybe instead of "flaws" you could say they're differences from RC

Reply:
not really problem anymore cuz i said what "flaws" means in intro